Chinese/大话西游.md

```diff
@@ -120,3 +120,4 @@
 
 [悟空](Goku/Guku_arrives.md)
 
+[测试](test/test.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.